packages/types/lib/scripts/syncs/api.ts

```diff
@@ -4,4 +4,4 @@ export interface SyncResult {
     deleted: number;
 }
 
-export type SyncType = 'INCREMENTAL' | 'FULL';
+export type SyncType = 'INCREMENTAL' | 'FULL' | 'WEBHOOK';
```

Comment:
`EXTERNAL_WEBHOOK` or maybe `THIRD_PARTY_WEBHOOK`? I feel like there needs to be some specificity since we're sending a webhook and then the `syncType` is webhook -- semi confusing

Reply:
it is true there are a lot of webhooks involved but I am not sure `sync type = external webhook` bring much more additional info